src/parseTools.js

```diff
@@ -1158,15 +1158,16 @@ function addReadyPromiseAssertions(promise) {
   // Also warn on onRuntimeInitialized which might be a common pattern with
   // older MODULARIZE-using codebases.
   properties.push('onRuntimeInitialized');
-  return properties.map((property) => {
-    const warningEnding = `${property} on the Promise object, instead of the instance. Use .then() to get called back with the instance, see the MODULARIZE docs in src/settings.js`;
-    return `
-      if (!Object.getOwnPropertyDescriptor(${promise}, '${property}')) {
-        Object.defineProperty(${promise}, '${property}', { configurable: true, get: function() { abort('You are getting ${warningEnding}') } });
-        Object.defineProperty(${promise}, '${property}', { configurable: true, set: function() { abort('You are setting ${warningEnding}') } });
-      }
-    `;
-  }).join('\n');
+  const warningEnding = ' on the Promise object, instead of the instance. Use .then() to get called back with the instance, see the MODULARIZE docs in src/settings.js';
+  const res = JSON.stringify(properties);
+  return res + `.forEach((prop) => {
+  if (!Object.getOwnPropertyDescriptor(${promise}, prop)) {
+    Object.defineProperty(${promise}, prop, {
+      get: () => abort('You are getting ' + prop + '${warningEnding}'),
+      set: () => abort('You are setting ' + prop + '${warningEnding}'),
```

Comment:
We do have a test for it yes: `other.test_assertions_on_ready_promise`.  It continues to pass.  `configurable` should only be needed if future modification to that property are to be allowed.. which doesn't seem necessary.

Reply:
Ok, if the test passes then sgtm.

Reply:
Because in the old code (for some reason) there are two different calls to `defineProperty` for each property.. one for `set` and one for `get`.  So the second call is mutating the existing property.

Reply:
Ah, thanks, now it makes sense...